### DISCUSSION TITLE
Исправление ошибки в Issue #159

### DISCUSSION THREAD

**Discussion by yezhakov** [malikzh/NCANode#179] [Show and tell]
Привет. Исправил ошибку из Issue #159. Каким образом можно стать контрибьютером и сделать пул-реквест?

**Comment by malikzh** [comment]
Приветствую, благодарю за pull request.
Я его смержил и сделал релиз.

вт, 10 окт. 2023 г. в 08:30, yezhakov ***@***.***>:

> Привет. Исправил ошибку из Issue #159
> <https://github.com/malikzh/NCANode/issues/159>. Каким образом можно
> стать контрибьютером и сделать пул-реквест?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/malikzh/NCANode/discussions/179>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIKJ3TO7VS7XM5MJBE3ZQDTX6U5U3AVCNFSM6AAAAAA52IZ4WWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVG4ZDCMJUGY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
С уважением, Malik Zharykov.
*Software Engineer*
E-mail: ***@***.***
GitHub: https://github.com/malikzh
Twitter: https://twitter.com/malikz_h
LinkedIn: https://www.linkedin.com/in/malik-zharykov-bba687165/
